### ISSUE TITLE
buy/sell limitation

### ISSUE THREAD

**Issue by doncat99** [timeolord/Reinforcement-Learning-Stock-Trader#3]
A limitation is that the algorithm allows processing buy/sell one stock at each step, but not manipulates multi-stock buy/sell at one step. And a guess is that such limitation would make the module learning process much harder.

**Comment by timeolord** [comment]
Yes that is a good point, however the main issue I run into with allowing multiple sell/buy stocks for each step is the size of the multidiscrete options increased too much. Since it uses MCST adding alot of possible outputs really slows down the training and prediction times. I tried with increasing the amount of positions it can hold for each ticker and just with 10 position per stock and 14 different tickers (3x10x14=420 permutations) it took over 8s to train each time step, and that only had a MCST search depth of 4 nodes. Also I think MCST is more important to the model than being able to sell multiple positions at once, especially since it's trained on hourly data so it isn't trading high frequency to begin with.

**Comment by doncat99** [comment]
Agree. What we wish our RL models to learn from the original data we feeding to, is the main issue. And I can't tell if multi-stock single model is better than single-stock multi models.
